ngraph_bridge/ngraph_utils.cc

```diff
@@ -32,6 +32,8 @@
 #include "ngraph_bridge/ngraph_utils.h"
```

Comment:
`ngraph_utils.*` is a kitchen sink already. I think that it's preferable to have helper methods such as these in the proper ngraph_bridge namespace instead of putting them all in utils.